srcpkgs/tdacli/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'tdacli'
+pkgname=tdacli
+version=0.3.0
+revision=1
+build_style=cargo
+makedepends="pkg-config openssl-devel"
+short_desc="TDAmeritrade API client"
+maintainer="Jason Manley <jason@jasondavid.tv>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/jbertovic/tdacli/"
+distfiles="https://codeload.github.com/jbertovic/${pkgname}/tar.gz/refs/tags/v${version}>${pkgname}-${version}.tar.gz"
```

Comment:
you shouldn't need to rename the file, and you should use the pre-redirected url:

```suggestion
distfiles="https://github.com/jbertovic/tdacli/archive/refs/tags/v${version}.tar.gz"
```